documentation/RLC-quickstart.md

```diff
@@ -16,96 +14,100 @@ If you are the first time to prepare the SDK, please follow the Azure SDK guidan
    The generated libraries should be in the [azure-sdk-for-js](https://github.com/Azure/azure-sdk-for-js) repo, so fork and clone it in your local then the absolute path is called **${SDK_REPO_ROOT} folder**.
 
 1. Project Folder Structure.  
-   Normally, the folder structure would be something like `sdk/{servicename}/{servicename}-{modulename}-rest`. For example, we have `sdk/agrifood/agrifood-farming-rest` folder for Farmbeats account modules under {SDK_REPO_ROOT}. That folder will be your **${PROJECT_ROOT} folder**.  
+   Normally, the folder structure would be something like `sdk/{servicename}/{servicename}-{modulename}-rest`. For example, we have `sdk/agrifood/agrifood-farming-rest` folder for Farmbeats account modules under {SDK_REPO_ROOT}. That folder will be your **${PROJECT_ROOT} folder**.
 1. Package Name Convention.  
    The package name for RLC is something like `@azure-rest/{servicename}-{modulename}`. For example, the package name for Farmbeats module is `@azure-rest/agrifood-farming`.
 
-
 # How to generate RLC
 
 We are working on to automatically generate everything right now, but currently we still need some manual work to get a releasable package. Here're the steps of how to get the package.
 
-1. **Add TypeScript emitter dependency in package.json**  
-   
-   In TypeSpec project, modify `package.json` to add dependency for TypeScript emitter, then run `npm install` again to install `@azure-tools/typespec-ts`.
-
-   ```json
-   "dependencies": {
-     ...
-     "@azure-tools/typespec-ts": "latest"
-   },
+1. \*_Configure python emitter in tspconfig.yaml_
+
+   In [Azure/azure-rest-api-specs(https://github.com/Azure/azure-rest-api-specs) repo, there should be tspconfig.yaml where main.tsp of your service is. Make sure it is configured for JavaScript SDK for example:
+
+   ```yaml
+   parameters:
+     "js-sdk-folder":
+       default: "{project-root}/azure-sdk-for-js/"
+     "service-directory-name":
+       default: "YOUR_SERVICE_DIRECRORY"
+   emit:
+     - "@azure-tools/typespec-ts"
+   options:
+     "@azure-tools/typespec-ts":
+       title: SERVICE_NAME
+       emitter-output-dir: "{js-sdk-folder}/sdk/{service-directory-name}/YOUR_PACKAGE_NAME-rest"
```

Comment:
I notice that we have a `/sdk/` defined so I was wondering what the `service-directory-name` value we expected is e.g: sdk\anomalydetector\ai-anomalydetector-rest or anomalydetector\ai-anomalydetector-rest

Reply:
service directory should be consistent for all emitters (see https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml#L11) As for the sdk in the path I think that is needed as well because this should point to the exact path in the repo.